.travis.yml

```diff
@@ -46,6 +40,5 @@ script:
         cargo test -p prometheus-static-metric --no-default-features --features="$FEATURES"
 
         # Run benchmarks.
-        cargo bench --no-default-features --features="$FEATURES"
         cargo bench -p prometheus-static-metric --no-default-features --features="$FEATURES"
```

Comment:
I am a bit reluctant to only partially move to criterion. Is there something blocking us from using criterion in prometheus-static-metric as well?

Reply:
Maybe because `prometheus-static-metric` itself is something that requires nightly>

Reply:
No sorry, I just ran out of time on Friday and preferred to split the move into smaller bits. I saw today that there wasn't too much left to converted, so I amended this PR now to complete the move.